<launch>

<!-- Starting Rover5 node -->
   <rosparam file="$(find euclid_rover5_pkg)/config/rover5.yaml" command="load"/>

   <node name="rover5" pkg="euclid_rover5_pkg" type="rover5_node" output="screen" />

   <node pkg="tf" type="static_transform_publisher" name="map_to_odom_publisher" args="0.0 0.0 0.0 0 0 0 1 map odom 100" />


   <!-- node pkg="tf" type="static_transform_publisher" name="map_to_footprint_publisher" args="0.0 0.0 0.0 0 0 0 1 map base_footprint 100" / -->
   <!-- node pkg="tf" type="static_transform_publisher" name="footprint_to_base_publisher" args="0.0 0.0 0.03 0 0 0 1 base_footprint base_link 100" / -->
   <!-- node pkg="tf" type="static_transform_publisher" name="base_to_camera_publisher" args="0.1175 0.0 0.1050 0 0 0 1 base_link camera_link 100" / -->

</launch>
